lib/decorators/deadline.js

```diff
@@ -0,0 +1,34 @@
+const { get } = require('lodash');
+const moment = require('moment-business-time');
+const Case = require('@ukhomeoffice/taskflow/lib/models/case');
+
+module.exports = settings => {
+  return c => {
+    if (get(c, 'data.model') !== 'project') {
+      return c;
+    }
+
+    const isAmendment = get(c, 'data.modelData.status') === 'active';
+
+    if (isAmendment) {
+      return c;
+    }
+
+    return Promise.resolve()
+      .then(() => Case.find(c.id))
```

Comment:
we re-fetch the case to get the activity log, which won't be available on `c` if a list of tasks is being decorated.